src/utils/asar.ts

```diff
@@ -0,0 +1,111 @@
+// @ts-ignore
```

Comment:
I understand that you have to put this since the `asar` package doesn't come with TS definitions, but it's still better to add an explaining comment here.